.bowerrc

```diff
@@ -0,0 +1,6 @@
+{
+  "directory": "bower_components",
+  "scripts": {
+    "postinstall": "gulp inject"
```

Comment:
This bit means that when you add a new bower dependency it will automatically inject it into index.html :smile: